src/components/hero.module.css

```diff
@@ -0,0 +1,43 @@
+html {
+    scroll-behavior: smooth;
+}
+
+.hero {
+    align-items: center;
+    color: white;
+    display: flex;
+    flex-flow: column nowrap;
+    justify-content: center;
+    min-height: 100vh;
+    margin-top: -155px;
+    height: 100px;
+    padding: 0;
+    text-align: left;
+    color: black;
+  }
+  
+  .content {
+    margin-bottom: 60px;
+  }
+
+  .h1 {
+    font-size: 64px;
+    font-weight: bold;
+  }
+  
+  p {
+    font-size: 18px;
+  }
+
+  .arrow {
```

Comment:
it'd be nice if the arrow were rounded. i think maybe this can be done by using an icon:
https://fontawesome.com/icons/chevron-down?style=solid
https://medium.com/@johnny02/how-to-add-font-awesome-to-a-gatsby-site-89da940924d5